build.gradle

```diff
@@ -0,0 +1,159 @@
+import java.text.SimpleDateFormat
+
+buildscript {
+    ext {
+        springBootVersion = '1.5.2.RELEASE'
+    }
+    repositories {
+        mavenLocal()
```

Comment:
This line ensures gradle to check one's local repository before reaching out to the external repository.

Reply:
I get having it below, for pulling dependencies, but wouldn't we only need it up here if the build script itself was pulling dependencies (like gradle plugins) from local builds?

Reply:
I defer to your judgement on how to proceed.  I was thinking that Gradle worked much like Maven which it will look at the local .m2 instead of pulling down the dependency each time a project is run.  I admit my lack of knowledge in this particular block so if that isn't advised I think we should remove it.

Can you just respond that you want me to remove it if it is there in error?

Reply:
I am about 95% sure it should be removed :-)

Reply:
Cool.  I'll remove it.  I'm still catching up on the Gradle from Maven move (on my part).